business/istio_config.go

```diff
@@ -176,7 +200,11 @@ func (in *IstioConfigService) GetIstioConfigMap(ctx context.Context, criteria Is
 	// Check if user has access to the namespace (RBAC) in cache scenarios and/or
 	// if namespace is accessible from Kiali (Deployment.AccessibleNamespaces)
 	for cluster := range in.userClients {
-		singleClusterConfigList, err := in.GetIstioConfigListPerCluster(ctx, criteria, cluster)
+		if criteria.Cluster != "" && cluster != criteria.Cluster {
```

Comment:
This condition is correct, but causes regression in another place, listing Apps does not load Istio Configs for other clusters than default.
PR sent https://github.com/jshaughn/kiali/pull/18

Reply:
Thanks for the PR, Hayk I merged it in.  See my comment in the other PR...